Source/FullScreenPokemon.ts

```diff
@@ -9384,6 +9385,7 @@ module FullScreenPokemon {
          */
         checkForOldStorageData(): void {
             if (!this.ItemsHolder.getItem("oldLocalStorage") || this.ItemsHolder.getItem("gameStarted")) {
+                this.ItemsHolder.getItem("collectionKeys");
```

Comment:
Not sure how this worked before... it might have been me not using completely new saves, but this has been removed. This was supposed to be the first point where `collectionKeys` was referenced so the value that was in localStorage was in `ItemsHolder.items`.